src/Microsoft.HttpRepl.Tests/Commands/PatchCommandTests.cs

```diff
@@ -1,94 +1,160 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Text;
+using System.Collections.Generic;
+using System.IO;
```

Comment:
I don't think this using is used anymore

Reply:
Removed it here:
https://github.com/aspnet/HttpRepl/commit/41e90b3a9ce64cf9eafa0e461e37e7bbb4cf979c